tests/test_api_long.py

```diff
@@ -143,6 +145,72 @@ def test_schema_with_project(self):
             self.assertTrue('user' in schema)
             self.assertTrue('visible' in schema['user'])
 
+# Requires the Shotgun instance to have transcoding and 'transcoder_malware_support' enabled.
+class TestMalwareScanning(base.LiveTestBase):
```

Comment:
Should these tests be in the workflow-tests?

Reply:
definitely! 
Lets move these tests to the workflow tests!

Reply:
Will do.